hypothesis-python/tests/cover/test_float_nastiness.py

```diff
@@ -244,4 +248,7 @@ def test_no_single_floats_in_range():
     high = low + 2
     st.floats(low, high).validate()  # Note: OK for 64bit floats
     with pytest.raises(InvalidArgument):
+        """Unrepresentable bounds are deprecated; but we're not testing that
+        here."""
+        warnings.simplefilter('ignore')
```

Comment:
This will change the filter globally - add the [`catch_warnings` context manager to make it temporary](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings).  
(no need to record or check warning type though)